shell/packages/sandstorm-db/db.js

```diff
@@ -153,6 +153,7 @@ Packages = new Mongo.Collection("packages");
 //   url:  When status is "download", the URL from which the SPK can be obtained, if provided.
 //   isAutoUpdated: This package was downloaded as part of an auto-update. We shouldn't clean it up
 //     even if it has no users.
+//   isPreinstalled: This package was downloaded in order to be pre-installed by users.
```

Comment:
This field is somewhat counter-intuitive as it really only flags why the package was originally downloaded, not whether the package is currently chosen for pre-installation. Would it make sense to check the admin setting dynamically inside the installer code, rather than check for this field on the package record?